### ISSUE TITLE
End of support! We need maintainers!

### ISSUE THREAD

**Issue by lowpolyv2** [JDALab/film-central#5]
Dear users,

We are no longer able to maintain the following plugins going forward:
- `film-central`
- `otaku-watcher`

The both of us no longer have the time to allocate to these projects. Just like we've been stating in our readme's we call for some new maintainers who could possibly add some new scrapers.

We've had word that mov-cli will start producing documentation around plugin development, so we hope that will assist the community with creating new plugins and scrapers just like `film-central` and `otaku-watcher`.

**Comment by zxxtlz** [comment]
🫡

**Comment by Edvinest** [comment]
I would like to maintain `film-central`, or should I just make my own plugin?

**Comment by lowpolyv2** [comment]
> I would like to maintain `film-central`, or should I just make my own plugin?

Whatever you prefer, I wouldn't mind either way. If you create a new one I can archive this and redirect everyone to yours. Also if you need any help I'm willing to help / contribute when I can.

**Comment by Edvinest** [comment]
> > I would like to maintain `film-central`, or should I just make my own plugin?
> 
> Whatever you prefer, I wouldn't mind either way. If you create a new one I can archive this and redirect everyone to yours. Also if you need any help I'm willing to help / contribute when I can.

I still have a week of University left, but after that I'd happily maintain this project. Where could we talk about the details?

**Comment by LuaMann** [comment]
> > > I would like to maintain `film-central`, or should I just make my own plugin?
> > 
> > 
> > Whatever you prefer, I wouldn't mind either way. If you create a new one I can archive this and redirect everyone to yours. Also if you need any help I'm willing to help / contribute when I can.
> 
> I still have a week of University left, but after that I'd happily maintain this project. Where could we talk about the details?

Not all heroes wear capes.

**Comment by lowpolyv2** [comment]
> I still have a week of University left, but after that I'd happily maintain this project. Where could we talk about the details?

Yeah sure take your time. I would prefer to talk about any details over GitHub.

Right now all we need for this plugin is a new and more stable scraper. You can have a look at our current scrapers and plugins or even [mov-cli's official plugins](https://github.com/orgs/mov-cli/repositories) to help understand how plugins are created. Or if you would like to create your own plugin from scratch there's even a video tutorial now that one of the devs at mov-cli made. https://youtu.be/2I84JHQRTfQ

**Comment by acollyte** [comment]
what's the status on this? i'm unable to use 2/3 of the sources made available due to this. will provide more details if needed.

affected scrapers are: vidsrcme and vidsrcto

**Comment by Edvinest** [comment]
> what's the status on this? i'm unable to use 2/3 of the sources made available due to this. will provide more details if needed.
> 
> affected scrapers are: vidsrcme and vidsrcto

I tried looking for reliable scrapers, with no luck as of now.

**Comment by acollyte** [comment]
i'm not an extremely confident software engineer but if you need testers or just someone to spitball with or help you with recruiting new volunteers, feel free shoot me a chatroom and let's consolidate our resources on this

**Comment by Edvinest** [comment]
Sorry to disappoint everybody, but I won't be able to maintain the project. I shot way past my goalpoast.

**Comment by lowpolyv2** [comment]
alright no worries, thank for trying. :heart:

**Comment by jansen44** [comment]
Hey, Idk if you guys are still looking for maintainers, but I'd gladly help! Just lmk, feel free to email me if needed.

**Comment by jansen44** [comment]
Sure, how you guys want to do it? I think I can by now just fork the project and start working on the "more stable scraper" part, after that I can open a PR to this repo and if you guys think it makes sense we can discuss more about it. Does that make sense?

**Comment by lowpolyv2** [comment]
> Sure, how you guys want to do it? I think I can by now just fork the project and start working on the "more stable scraper" part, after that I can open a PR to this repo and if you guys think it makes sense we can discuss more about it. Does that make sense?

Yep sounds great. 👍 Ping us anywhere here on GitHub if you are unsure of anything or need a bit of help.

**Comment by zxxtlz** [comment]
I don't know how this scraper thing works but i found something, hope its helpful to you guys
https://fmhy.pages.dev/internet-tools#web-scraping-crawling

**Comment by jansen44** [comment]
> > Sure, how you guys want to do it? I think I can by now just fork the project and start working on the "more stable scraper" part, after that I can open a PR to this repo and if you guys think it makes sense we can discuss more about it. Does that make sense?
> 
> Yep sounds great. 👍 Ping us anywhere here on GitHub if you are unsure of anything or need a bit of help.

nice, I'll probably start this week
